### PR TITLE
DIRECTOR: Add more games, fix some entries

### PR DESCRIPTION
**Pull request by einstein95**
Additionally correct a few whitespace issues (tabs following spaces) and at least one spelling mistake

### CODE REVIEW

**Review by trembyle — CHANGES_REQUESTED** [review]
@einstein95 let me know if and when this is ready to be reviewed.

If you'd like to keep it open until then, please move it into Draft state. If you think that you're going to just resubmit it, then we should close it.

**Comment by einstein95** [comment]
@trembyle Ready to be reviewed now, I'll add a 2nd file for `milo` and `milomagical` now

File: engines/director/detection_tables.h [hunk 1]
```diff
@@ -327,6 +340,7 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "searchlearn",     	"Search & Learn Adventures" },
 	{ "secretkeys",     	"Search for the Secret Keys with Mickey" },
 	{ "secretpaths1",		"Secret Paths in the Forest" },
+	{ "secretsluxor",		"Secrets of the Luxor" },
```
Comment (trembyle) [inline engines/director/detection_tables.h:343]: Can you make this just `luxor`? (sorry, just a preference)

File: engines/director/detection_tables.h [hunk 2]
```diff
@@ -1740,11 +1811,10 @@ static const DirectorGameDescription gameDescriptions[] = {
...
 #define SUPPORT_STATUS ADGF_NO_FLAGS
 
-	MACGAME1("lzone", "",   "L-ZONE", 		"f5277c53bacd27936158dd3867e587e2", 392484, 300),
-	MACGAME1("lzone", "v2", "L-ZONE", 		"276bee761e48a6fd709df77d5c2f60dd", 395344, 300),
-	GENGAME1_("lzone", "",	"L-ZONE",		"9f0bb7ec7720e4f680ee3aa3d22c1c9d", 384968, Common::EN_ANY, Common::kPlatformMacintoshII,ADGF_MACRESFORK, 300),
-	WINGAME2("lzone", "",   "L_ZONE.EXE",	"65d06b5fef155a2473434571aff5bc29", 370009,
-							"SYNER_01.MMM", "56b6f1c68e85a96bcdd01028bdec2d35", 460594, 300),
+	MACGAME1("lzone",  "",   "L-ZONE",		"f5277c53bacd27936158dd3867e587e2", 392484, 300),
+	MACGAME1("lzone",  "v2", "L-ZONE",		"276bee761e48a6fd709df77d5c2f60dd", 395088, 310),
+	GENGAME1_("lzone", "",   "L-ZONE",		"9f0bb7ec7720e4f680ee3aa3d22c1c9d", 384968, Common::EN_ANY, Common::kPlatformMacintoshII,ADGF_MACRESFORK, 300),
```
Comment (trembyle) [inline engines/director/detection_tables.h:1816]:
```suggestion
	GENGAME1_("lzone", "",   "L-ZONE",		"9f0bb7ec7720e4f680ee3aa3d22c1c9d", 384968, Common::EN_ANY, Common::kPlatformMacintoshII, ADGF_MACRESFORK, 300),
```

File: engines/director/detection_tables.h [hunk 3]
```diff
@@ -1821,10 +1895,15 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINDEMO1("mspublishd",	   "Demo",		"PUBLISH.EXE",  "2b3543a9131a49f665982d26513a84f8", 1644502, 310),
 	WINDEMO1("mssndbits",	   "Demo",		"SNDBITS.EXE",  "2b3543a9131a49f665982d26513a84f8", 1476769, 310),
 
+	MACGAME1("msdinosaurs", "Overview Movie", "Overview Movie", "479541070b3cbab6fd8473ce6267fb98", 792604, 310),
+
 	MACDEMO1("msproject", "Demo", "MS Project demo", "9f0bb7ec7720e4f680ee3aa3d22c1c9d", 1060170, 300),
 
+	// Message from Apple found on Nikkei Mac CD-ROM (1994-2-17)
 	MACGAME1_l("msgfromapple", "", "Message", "4c7fddccfe5ab2e9c6398efa5dbe37c1", 395782, Common::JA_JPN, 313),
+	MACGAME1_l("msgfromapple", "", "Message from Apple", "4c7fddccfe5ab2e9c6398efa5dbe37c1", 395566, Common::JA_JPN, 313),
```
Comment (trembyle) [inline engines/director/detection_tables.h:1904]: I'll check my files. It's possible that there is a different filename on different distributions.

File: engines/director/detection_tables.h [hunk 4]
```diff
@@ -1901,7 +1985,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	MACDEMO1("pixar", "1993 Demo", "Pixar Projector", "1ae45c23586b41997ba52e2e7c771c4c", 2324198, 310),
 
-	MACDEMO1("playroom", "Color Demo", "Color Playroom Demo", "b29b31f423a332e7f9e059187e62e1be", 1000734, 300),
+	MACDEMO1("playroom", "Color Demo", "Color Playroom Demo", "b29b31f423a332e7f9e059187e62e1be", 1000478, 300),
+	// MACDEMO1("playroom", "Demo",	   "Demo", 				  "b29b31f423a332e7f9e059187e62e1be", 1000478, 300),
```
Comment (trembyle) [inline engines/director/detection_tables.h:1989]: Is this meant to be commented? If not, can you add a second file?

File: engines/director/detection_tables.h [hunk 5]
```diff
@@ -1930,7 +2017,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1_l("refixion3", "", "The Reindeer Story", "f5277c53bacd27936158dd3867e587e2", 392473, Common::JA_JPN, 300),
 
 	// Original Mac version is Director. Later DOS port uses MADE engine (also in ScummVM).
-	MACDEMO1("rodneyfs", "Demo", "Rodney's Funscreen demo", "9f0bb7ec7720e4f680ee3aa3d22c1c9d", 356866, 300),
+	MACDEMO1("rodneyfs", "Demo", "Rodney's Funscreen demo", "9f0bb7ec7720e4f680ee3aa3d22c1c9d", 356610, 300),
+	// MACDEMO1("rodneyfs", "Demo", "Sample Application",	"9f0bb7ec7720e4f680ee3aa3d22c1c9d", 356610, 300),
```
Comment (trembyle) [inline engines/director/detection_tables.h:2021]: Why is this one commented?

File: engines/director/detection_tables.h (hunk 4 above)
Comment (einstein95) [reply engines/director/detection_tables.h:1989]: That file is the only one in the folder and is identical to the existing demo

File: engines/director/detection_tables.h (hunk 5 above)
Comment (einstein95) [reply engines/director/detection_tables.h:2021]: Similarly to the playroom demo above, the genericness of the filename and it being identical to the existing demo is why I commented it

File: engines/director/detection_tables.h (hunk 1 above)
Comment (einstein95) [reply engines/director/detection_tables.h]: Done

File: engines/director/detection_tables.h (hunk 2 above)
Comment (einstein95) [reply engines/director/detection_tables.h]: Fixed

**Pull request merged**